src/main/resources/assets/oritech/lavender/entries/oritech_guide/interaction/enderic_laser.md

```diff
@@ -54,6 +54,21 @@ The laser ignores all input and output limits, and can fill the energy storage o
 
 ;;;;;
 
+**Machine Hunter Addon**
+
+When the enderic laser has a machine hunter addon, it will target mobs instead of blocks. Addon upgrades still work when hunting mobs: the yield addon will make the
+mobs drop more loot, the more hunter addons will extend the target scan range, the crop filter will avoid killing baby animals.
```

Comment:
Maybe also mentioning that more energy transferred (e.g. speed addons) will increase the damage dealt

Reply:
I updated the manual to fix the bolded header and mention the affect of speed addons.